docs/_tutorials/read-only-json-api.md

```diff
@@ -0,0 +1,321 @@
+---
+title: Read-Only JSON API
+author: izdwuut
+date: 2020-12-23 10:04:00 +0100
+---
+
+There shouldn't be a necessity to explain what is the [API](https://en.wikipedia.org/wiki/API). I assume that you've landed here to learn it in a context of Jekyll. Let's jump straight to the actual thing, shall we?
+
+A huge inspiration to write this tutorial was a [post](https://forestry.io/blog/build-a-json-api-with-hugo/) written by [Régis Philibert](https://forestry.io/authors/r%C3%A9gis-philibert/). It's about doing the exact same thing in [Hugo](https://gohugo.io/).
+
+I hope that my solution in Jekyll is elegant, too. You just need to define some [layouts]({% link _docs/layouts.md %}) and write a [plugin]({% link _docs/plugins.md %}) to convert your content to JSON. **Don't be afraid if you're not familiar with Ruby** --- the language that Jekyll is written in. It's similar to Python, and you'll get a hang of it quickly.
+
+By the end of the tutorial, you'll expose the following API:
+* `/` --- an index of all posts
+* `/{url}` --- details of a post
+* `/categories` --- an index of categories
+* `/categories/{category}` --- an index of posts under a category
+
+Let's dive in!
+
+---
+
+# Setup
+
+First, you need to go through an [installation]({% link _docs/installation.md %}) process. Having that, you can initialize a new Jekyll project. In your directory of choice, invoke the following:
+
+```sh
+bundle init
+bundle config set --local path 'vendor/bundle'
+bundle add jekyll
+bundle exec jekyll new --force --skip-bundle .
+bundle install
+```
+
+* Initialize a new Ruby [bundle](https://bundler.io).
+* Instruct Bundler to install dependencies locally.
+* Add Jekyll as a dependency.
+* Create a new scaffold for the project. 
+* Install dependencies.
+
+Now, you need to do some clean up. 
+
+---
+
+# Configuration
```

Comment:
Rather increase `#` to `##`. And existing `##` to `###`.

Having multiple H1 headings on an HTML page is not a good pattern for styling, accessibility and probably SEO. It is also not consistent with other pages like [navigation](https://raw.githubusercontent.com/jekyll/jekyll/1d4cff612bcfe28b5dc1be8331b6a47bc3b909f1/docs/_tutorials/navigation.md) tutorial which uses H2s, no H1 and doesn't use `---` to split sections.

Reply:
Done. Wasn't aware of that.